### PR TITLE
[sailfish-browser] Add fader on top of the web content. Contributes to JB#29100

### PR DESCRIPTION
**Pull request by rainemak**
Fader is used to reduce flickering during orientation change. It fades
in with background color of the web page when rotation transition
starts. At the end of the orientation change transition pauses for 700ms
giving web engine a bit of time to relayout.

Content orientation is updated to the browser content window immediately
when fader has faded in.

### CODE REVIEW

File: src/pages/components/OrientationFader.qml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+    property alias orientationTransition: transition
+    property alias page: propertyAction.target
+    property alias fadeTarget: fadeOut.target
+    readonly property alias running: transition.running
+
+    signal applyContentOrientation
+
+    anchors.fill: parent
+    opacity: 0.0
```
Comment (rainemak) [inline src/pages/components/OrientationFader.qml]: This can be binding.

`opacity: running ? 1.0 : 0.0`

**Comment by rainemak** [comment]
Updated.

**Comment by rainemak** [comment]
Rebased

**Comment by rojkov** [comment]
LGTM. Seems to work

**Comment by tworaz** [comment]
LGTM

**Pull request merged**